warcprox/warcproxy.py

```diff
@@ -199,6 +199,10 @@ def _parse_warcprox_meta(self):
     def _proxy_request(self):
         warcprox_meta = self._parse_warcprox_meta()
         remote_ip = self._remote_server_conn.sock.getpeername()[0]
+        if remote_ip == "127.0.0.1":
```

Comment:
```suggestion
        if remote_ip == "127.0.0.1" or remote_ip == "::1":
```

This adds IPv6 support. On my system, `localhost` appears to resolve this way.

Reply:
Done